README.md

```diff
@@ -343,17 +341,18 @@ This is how we create and publish a new release of the 121-platform.
 
 ### Patch/Hotfix Checklist
 
-This follows the same process as a regular release + deployment. With some small changes.
+This follows a similar process to regular release + deployment, with some small changes.
 
-- Code does not need to be frozen. (As there is no active development on the release-branch)
-
-- Checkout the `release/<version>`-branch that needs the hotfix.
-- Create a new local branch on top of it (e.g. `release/<v1.x.1>`) and make the changes
+- Code does not need to be frozen.
+- Checkout the `<version>` tag which contains the code that you want to hotfix from.
+- Create a new local hotfix branch using that tag as the head (e.g. `hotfix/<v1.x.1>`) and make the changes
 - Add the hotfix-release to the [CHANGELOG](CHANGELOG.md)
 - Push this branch to the upstream/origin repository.
-- Create a new release (see above) and publish it.
+- Create a new release (see above) selecting the `hotfix` branch as target, and publish it.
 - Use the [deployment-workflows on GitHub Actions](https://github.com/global-121/121-platform/actions) to deploy to production (for each instance)
```

Comment:
Maybe redundant, but we could also mention here that the-previously-created TAG should be deployed... Not the branch.

Reply:
```suggestion
- Use the [deployment-workflows on GitHub Actions](https://github.com/global-121/121-platform/actions) to deploy the newly created tag (*not the branch*) to production (for each instance)
```